packages/cli-kit/src/public/node/environment.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * It returns the environment variables of the environment
+ * where the Node process is running.
+ *
+ * This function exists to prevent the access of the process
+ * global variable which is discouraged via the no-process-env
+ * ESLint rule.
+ *
+ * @returns Current process environment variables.
+ */
+export function getEnvironmentVariables(): NodeJS.ProcessEnv {
+  return {...process.env}
+}
```

Comment:
I very strongly approve of this strategy. Excellent for testing, and ensures we don't accidentally write to `process.env`.

Reply:
why not just `return process.env`?